### DISCUSSION TITLE
[Projects] Add new issues in a repository automatically to a project

### DISCUSSION THREAD

**Discussion by kuzdogan** [community/community#38849] [Projects and Issues]
### Select Topic Area

Product Feedback

### Body

We started using the new Projects, as we assume the Projects (classic) will be sunsetted soon.

For the project, we want to have the similar automation that was available to Projects (classic) ,which was on a repository level, that when an issue is opened on a repository, it will be added to the Project's in Todo status. 

Is this already somehow available or planned? If already there, I wasn't able to find it in docs or anywhere else. With the current "workflow" in the Projects, the issue will be added as "Todo" when it's added to the project **manually**.

**Comment by lukewar** [comment]
Hi @kuzdogan, it is something we are looking into. In the meantime please use GitHub Actions: https://github.com/actions/add-to-project

**Reply by kuzdogan** [reply]
Yes we are aware of the workaround but that needs further tweaking and access control https://github.com/actions/add-to-project/issues/264 

For now we'll be adding issues manually. You can go to your repo's Issues, select all issues and choose your project board under Projects in the table header.
